nodes/nodes/nodes-sno-worker-nodes.adoc

```diff
@@ -17,7 +22,7 @@ Adding worker nodes does not expand the cluster control plane, and it does not p
 
 [NOTE]
 ====
-Unlike multi-node clusters, by default all ingress traffic is routed to the single control-plane node, even after adding additional worker nodes.
+To add worker nodes, you must have access to the OpenShift Cluster Manager. This method is not supported when using the Agent-based installer to install a cluster in a disconnected environment.
```

Comment:
OpenShift Cluster Manage needs to be an attribute as per the common-attributes file.

Reply:
ack.